justfile

```diff
@@ -138,7 +138,11 @@ generate-config-schema:
     cargo xtask generate-config-schema > restate_config_schema.json
 
 check-deny:
-    cargo deny --all-features check
+    #!/usr/bin/env bash
+    # cargo-deny-action runs as a standalone workflow in CI
```

Comment:
That's a good one.

Reply:
Figured it's a better workaround to keep the existing `deps.yml` workflow in place, it's cleaner to use the `cargo-deny-action` there :-)